### PR TITLE
Fix app background svg provider crash

### PR DESCRIPTION
**Pull request by slawekkrol**
Fixes:
https://github.com/yang-f/flutter_svg_provider/issues/49

Steps to reproduce:
- app using flutter_svg_provider 
- go app to background
- logs show error:
The following _Exception was thrown resolving a single-frame
image stream:
Exception: operation failed
- go app to foreground
- all images using svg_provider are gone

This PR fixes this crash by using direct sync method

### CODE REVIEW

File: lib/flutter_svg_provider.dart [hunk 1]
```diff
@@ -111,20 +111,24 @@ class Svg extends ImageProvider<SvgImageKey> {
...
+            pictureInfo.size.height.round(),
+          )
+        : pictureInfo.picture.toImageSync(
+            pictureInfo.size.width.round(),
+            pictureInfo.size.height.round(),
+          );
 
     return ImageInfo(
       image: image,
-      scale: 1.0,
```
Comment (yang-f) [inline lib/flutter_svg_provider.dart:127]: Why was this removed?

File: lib/flutter_svg_provider.dart [hunk 2]
```diff
@@ -111,20 +111,24 @@ class Svg extends ImageProvider<SvgImageKey> {
   }
 
   static Future<ImageInfo> _loadAsync(SvgImageKey key) async {
-    final String rawSvg = await _getSvgString(key);
+    final rawSvg = await _getSvgString(key);
```
Comment (yang-f) [inline lib/flutter_svg_provider.dart:114]: What is the reason for the change here?

Comment (slawekkrol) [reply lib/flutter_svg_provider.dart:114]: Unnecessary type annotation on a local variable. ([Documentation](https://dart.dev/tools/linter-rules/omit_local_variable_types))

File: lib/flutter_svg_provider.dart (hunk 1 above)
Comment (slawekkrol) [reply lib/flutter_svg_provider.dart:127]: The value of the argument is redundant because it matches the default value. [(Documentation)](https://dart.dev/tools/linter-rules/avoid_redundant_argument_values)

**Review by yang-f — APPROVED** [review]
LGTM.
